Viper/Viper/Router/MainRouterProtocol.swift

```diff
@@ -0,0 +1,16 @@
+// MainRouterProtocol.swift
+// Copyright © Natasha Ananas. All rights reserved.
+
+/// Протокол роутера
+protocol MainRouterProtocol {
+    var builder: AssemblyBuilderProtocol? { get set }
+    func initialViewController()
+    func popToRoot()
+    func didSelectRowAt(country: Country)
+}
+
+/// Роутер инпут протокол
+protocol CountryRouterInputProtocol {
+    var assembly: AssemblyBuilderProtocol? { get set }
+    func goHotelViewController(index: Int)
+}
```

Comment:
```suggestion
/// Протокол роутера
protocol MainRouterProtocol {
    var builder: AssemblyBuilderProtocol? { get set }
    func initialViewController()
    func popToRoot()
    func didSelectRowAt(country: Country)
}

/// Роутер инпут протокол
protocol CountryRouterInputProtocol {
    var assembly: AssemblyBuilderProtocol? { get set }
    func goHotelViewController(index: Int)
}
```
может все же в разные файлы?